Fix Bug 1472209 - Remove StartupOverlay if the tour is skipped

piatra opened the pull request:
issue described in https://github.com/piatra/activity-stream/blob/1cb7444a7bd1c653e87512d5b9e4ce83e454c9b2/content-src/components/StartupOverlay/StartupOverlay.jsx#L62-L63

Mardak commented on content-src/components/StartupOverlay/StartupOverlay.jsx:
```diff
@@ -59,10 +59,16 @@ export class _StartupOverlay extends React.PureComponent {
     let privacyLink = (<a href="https://accounts.firefox.com/legal/privacy" target="_blank" rel="noopener noreferrer"><FormattedMessage id="firstrun_privacy_notice" /></a>);
     return (
-      <div className={`overlay-wrapper ${this.state.show ? "show " : ""}`}>
+      <div className="overlay-wrapper show">
```
Isn't the non-show overlay-wrapper needed so that there's the transition? @ericawright ?

piatra commented:
Good catch! I updated the PR: 
* the overlay still needs to removed, I deferred that to happen after the animation ends
* we need to prevent the `welcome` class name from being added when `updateTheme` is called because it hides the overflow
  * the observation here is that we only need to add it once so I moved it to `componentWillMount`
